src/Avalonia.FuncUI.Components/Wires.fs

```diff
@@ -0,0 +1,210 @@
+namespace Avalonia.FuncUI
+
+open System
+
+type IConnectable =
+    inherit IDisposable
+    abstract InstanceId: Guid with get
+
+type ITap<'signal> =
+    inherit IConnectable
+    abstract member CurrentSignal: 'signal with get
+    abstract member Subscribe : ('signal -> unit) -> IDisposable
+
+type IWire<'signal> =
+    inherit ITap<'signal>
+    abstract member Send : 'signal -> unit
```

Comment:
Not really happy with the naming here. Open for suggestions.

Reply:
> Initially considered this, but `IObservable` does not contain a value.

I had to look it up, but `IEvent` already inherits from `IObservable`, so it seems that the door is still open to allowing integration with Rx.